cornice/service.py

```diff
@@ -148,13 +155,17 @@ class Service(object):
     :meth:`~put`, :meth:`~options` and :meth:`~delete` are decorators that can
     be used to decorate views.
     """
-    renderer = 'simplejson'
+    renderer = 'cornicejson'
     default_validators = DEFAULT_VALIDATORS
     default_filters = DEFAULT_FILTERS
 
     mandatory_arguments = ('renderer',)
     list_arguments = ('validators', 'filters', 'cors_headers', 'cors_origins')
 
+    @classmethod
+    def configure_default_renderer(cls, renderer="simplejson"):
```

Comment:
no need for default value for `renderer` parameter

Reply:
right. comes from an earlier version of the patch